src/components/Thanks/MyKiva/OptInModule.vue

```diff
@@ -0,0 +1,253 @@
+	selectedLoan: {
+		type: Object,
+		default: () => ({})
+	},
+	loans: {
+		type: Array,
+		default: () => ([])
+	},
+	optedIn: {
+		type: Boolean,
+		default: false
+	}
+});
+
+const apollo = inject('apollo');
+const newConsentAnswered = ref(false);
+const receiveNews = ref(false);
+
+const { isMobile } = useIsMobile(MOBILE_BREAKPOINT);
+
+const borrowerName = computed(() => {
+	return props.selectedLoan?.name ?? '';
+});
+
+const title = computed(() => {
+	if (props.optedIn) {
+		return 'Thank you! You reached a milestone';
+	}
+	if (props.loans.length === 1) {
+		return `You and ${borrowerName.value} are in this together now.`;
+	}
+	if (props.loans.length === 2) {
+		const names = props.loans.map(loan => loan?.name).join(' and ');
+		return `Thank you! You, ${names} are in this together now.`;
+	}
+
+	return `Thank you! You, ${props.loans[0]?.name}, ${props.loans[1]?.name}
+		and ${props.loans[2]?.name} are in this together now.`;
+});
+
+const description = computed(
+	() => `Want to hear how you're impacting ${borrowerName.value}'s life and more ways to help people like them?`
+);
+
+const loansToDisplay = computed(() => {
+	const loans = [...props.loans];
+	if (props.loans.length === 3) {
+		const indexToRemove = loans.indexOf(loan => loan.id === props.selectedLoan.id);
+		const removedLoan = loans.splice(indexToRemove, 1)[0];
+		loans.splice(1, 0, removedLoan);
+	}
+	return loans.slice(0, 3);
+});
+
+const marginLeftWeight = computed(() => {
+	if (loansToDisplay.value.length === 1) {
+		return 0;
+	}
+	if (loansToDisplay.value.length === 2) {
+		return 6;
+	}
+
+	if (isMobile.value) {
+		return 7;
+	}
+
+	return 12;
+});
+
+const hash = loan => {
+	return loan?.image?.hash ?? '';
+};
+
+const updateOptIn = value => {
+	if (value) {
+		try {
+			apollo.mutate({
+				mutation: gql`
+					mutation updateCommunicationSettings(
+						$lenderNews: Boolean
+					) {
+						my {
+							updateCommunicationSettings(
+								communicationSettings: {
+									lenderNews: $lenderNews
+								}
+							)
+						}
+					}
+				`,
+				variables: {
+					lenderNews: value,
+				},
+			});
+		} catch (error) {
+			logReadQueryError(error, 'error');
+		}
+	}
+	newConsentAnswered.value = true;
+	receiveNews.value = value;
+};
+</script>
+
+<style lang="postcss" scoped>
+
+.module-container {
+	max-width: 620px;
+	max-height: 800px;
+
+	@apply tw-flex tw-flex-col tw-mx-auto tw-overflow-hidden tw-opacity-full tw-bg-white
+		tw-text-center tw-px-3 md:tw-px-8 tw-gap-3 tw-rounded-lg tw-py-4;
+}
+
+.borrower-container {
+	animation: fadein ease-in 1s;
+	width: 70px;
+
+	@screen md {
+		width: 150px;
+	}
+
+	@apply tw-block tw-relative tw-mx-auto tw-z-4;
+}
+
+.borrower-container > div {
+	height: 100px;
+
+	@screen md {
+		height: 200px;
+	}
+}
+
+.borrower-image {
+	width: 80px !important;
+	height: 80px !important;
+
+	@screen md {
+		width: 160px !important;
+		height: 160px !important;
+	}
+
+	box-shadow: '0px 4.42px 22.1px 0px #D1DCD6';
```

Comment:
This `box-shadow` isn't working (there's also an error in the CSS panel in Chrome). Use our existing drop shadow like in my recent PR.